cookbooks/aws-parallelcluster-shared/attributes/users.rb

```diff
@@ -10,6 +10,8 @@
 default['cluster']['slurm']['user_id'] = node['cluster']['reserved_base_uid'] + 1
 default['cluster']['slurm']['group'] = node['cluster']['slurm']['user']
 default['cluster']['slurm']['group_id'] = node['cluster']['slurm']['user_id']
+default['cluster']['slurm']['resume_group'] = 'resume-program'
```

Comment:
What about giving a more slurm related name, such as `slurm-resume` or `slurm-resume-program`?

Reply:
need to check what's the limit first. Will consider this, thanks